slices/xz-utils.yaml

```diff
@@ -0,0 +1,26 @@
+package: xz-utils
```

Comment:
Thanks for the contribution! The slice itself looks good to me. However could you add some spread tests to help us verify everything works as expected? https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr

Here is an example PR with tests:
https://github.com/canonical/chisel-releases/pull/320/files#diff-40e68e9abb4e657ef292dcfd6fa750ddffea20daec62f7ceb9acc7605beb17e3

Reply:
I suppose I don't need to add a test for every binary provided.

But when testing `xz` I thought I could test with `xz --version`, `xz -z` (compress) and a little `xzgrep` just for "completeness" of the test. But this failed and I can't understand why.

I'll send a fixup without the `xzgrep` that may pass, but I am not satisfied.